src/pages/HomePage.js

```diff
@@ -0,0 +1,166 @@
+import React, { useEffect, useState } from 'react';
+import { Circles } from 'react-loader-spinner';
+import { useDispatch, useSelector } from 'react-redux';
+import { FaArrowCircleRight } from 'react-icons/fa';
+import styled from 'styled-components';
+import { Link } from 'react-router-dom';
+import CoronaImg from '../images/corona.png';
+import { getContinents } from '../redux/continents';
+
+const HomePage = () => {
+  const { status, continents } = useSelector((state) => state.Continents);
+
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    dispatch(getContinents());
+  }, []);
+
+  const [plate, setPlate] = useState(false);
+
+  const handleOnChange = (e) => {
+    const textInput = e.target.value.toLowerCase() || '';
+    const str = textInput[0].toUpperCase().concat(textInput.slice(1, textInput.length))
+      || '';
+    setPlate(
+      continents.filter((continent) => continent.countries.includes(str)),
+    );
+  };
+
+  return (
+    <div className="continents">
+      <CoronaPic>
+        <Title>
+          Covid-19
+          <br />
+          Tracker
+        </Title>
```

Comment:
- [ ] The title with text **_Covid-19 Tracker_** is not properly positioned. I would advise you to reduce the covid image and position the text to the right just like in the guideline or you can decide to place it under the image.

### Your Design

![image](https://user-images.githubusercontent.com/77106962/181652167-1c8b3578-3137-46dc-9917-fe430e44998f.png)

### Design Template

![image](https://user-images.githubusercontent.com/77106962/181652290-fb79aaa2-7b2a-49c2-885b-6cfb15d81f12.png)